sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -335,6 +338,7 @@ export class EventHubClient {
    * @throws {AbortError} Thrown if the operation is cancelled via the abortSignal.
    */
   async getPartitionIds(abortSignal?: AbortSignalLike): Promise<Array<string>> {
+    throwErrorIfConnectionClosed(this._context);
```

Comment:
Should we add a @throws section to the JSDOC for this error? Maybe add something like "Create a new EventHubsClient and call getPartitionIds when encountering this error"